Dockerfiles/midstream/run_tests.py

```diff
@@ -23,7 +23,7 @@ def setUp(self):
                                         '/usr/local/bin/umoci')
 
     def run_extract_operator_bundle(self):
-        exec_cmd = "ansible-playbook -i localhost, -c local -v operator-test-playbooks/extract-operator-bundle.yml \
+        exec_cmd = "ansible-playbook -i localhost, -c local -vvvv operator-test-playbooks/extract-operator-bundle.yml \
```

Comment:
-1

Verbosity should be configurable.
And midstream expect configuration just with using env variables.